backend/open_webui/routers/pipelines.py

```
@@ -196,7 +190,7 @@ async def upload_pipeline(
     file: UploadFile = File(...),
     user=Depends(get_admin_user),
 ):
-    log.info(f"upload_pipeline: urlIdx={urlIdx}, filename={file.filename}")
+    log.debug(f"upload_pipeline: Processing file upload")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)